analyzers/its/expected/akka.net/Akka--netstandard2.0-S2326.json

```diff
@@ -0,0 +1,30 @@
+{
+"issues":  [
+{
+"id":  "S2326",
+"message":  "'T' is not used in the interface.",
+"location":  {
+"uri":  "sources\akka.net\src\core\Akka\Dispatch\IRequiresMessageQueue.cs",
+"region":  {
+"startLine":  18,
+"startColumn":  44,
+"endLine":  18,
+"endColumn":  45
+}
+}
+},
+{
+"id":  "S2326",
+"message":  "'TQueue' is not used in the interface.",
+"location":  {
+"uri":  "sources\akka.net\src\core\Akka\Dispatch\Mailbox.cs",
```

Comment:
False positives. See below.

Reply:
As discussed offline, we'll declare it as TP